ledger/sandbox/src/main/scala/com/digitalasset/platform/apiserver/services/admin/ApiConfigManagementService.scala

```diff
@@ -44,7 +44,8 @@ final class ApiConfigManagementService private (
 
   private val logger = ContextualizedLogger.get(this.getClass)
 
-  private val defaultConfigResponse = configToResponse(ledgerConfiguration.initialConfiguration)
+  private val defaultConfigResponse = configToResponse(
+    ledgerConfiguration.initialConfiguration.copy(generation = LedgerConfiguration.NoGeneration))
```

Comment:
Is it sensible to return a config, even if no config is actually set? I see that you need to return a generation, but the model could in principle be left empty. Would this break other tests?

Reply:
I agree, but this would be a bigger change and I suggest we leave it until after we release. Perhaps @rautenrieth-da can weigh in.

Reply:
:+1:

Reply:
https://github.com/digital-asset/daml/issues/6668